### ISSUE TITLE
RuntimeError: ApiV2 JSON seems to be invalid

### ISSUE THREAD

**Issue by ilf** [jaylinski/kodi-addon-soundcloud#57]
I installed kodi-addon-soundcloud 4.0.1 on Kodi 19.3. Some things work, other's don't.

Trying to open a Soundcloud artist playlist results in this error:

`GetDirectory - Error getting plugin://plugin.audio.soundcloud/?action=call&call=%2fusers%2f$redactedsoundcloudid%2fplaylists_without_albums`

```
EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
 - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
Error Type: <class 'RuntimeError'>
Error Contents: ApiV2 JSON seems to be invalid
Traceback (most recent call last):
  File "/home/osmc/.kodi/addons/plugin.audio.soundcloud/addon.py", line 3, in <module>
    plugin.run()
  File "/home/osmc/.kodi/addons/plugin.audio.soundcloud/resources/plugin.py", line 44, in run
    collection = listItems.from_collection(api.call(args.get("call")[0]))
  File "/home/osmc/.kodi/addons/plugin.audio.soundcloud/resources/lib/soundcloud/api_v2.py", line 82, in call
    return self._map_json_to_collection(res)
  File "/home/osmc/.kodi/addons/plugin.audio.soundcloud/resources/lib/soundcloud/api_v2.py", line 220, in _map_json_to_collection
    raise RuntimeError("ApiV2 JSON seems to be invalid")
RuntimeError: ApiV2 JSON seems to be invalid
-->End of Python script error report<--
```

**Comment by ilf** [comment]
@jaylinski Thanks, looking forward to the release.

**Comment by jaylinski** [comment]
@ilf Version 4.0.2 is released, does it work for you now?

**Comment by ilf** [comment]
@jaylinski I had already patched my old version, but yes 4.0.2 works too. Thanks a lot!
